logic/broadcaster.js

```diff
@@ -137,7 +129,6 @@ class Broadcaster {
 	 */
 	broadcast(params, options, cb) {
 		params.limit = params.limit || this.config.peerLimit;
```

Comment:
`self.config.peerLimit`

Reply:
Regarding that default limit. Would be nice if we can get rid of the condition that is used below:
- when `params.limit` is equal to `self.config.peerLimit` then truncate `peers` to `self.config.broadcastLimit`, then truncate to `self.config.peerLimit`
- when above condition is not met just truncate to `self.config.peerLimit`

It's unclear and can result with unexpected behavior.
Also we're returnign `body` property from this function, for what?

Reply:
WTF yes that's really bad logic ;p